### ISSUE TITLE
Pointers not being removed/cancelled using TUIO input

### ISSUE THREAD

**Issue by raymond9991** [TouchScript/TouchScript#356]
Hi, my use case is I use TUIO Input with a sensor.

I noticed that when the number of pointers keep increasing, sometimes the pointers will just clump up and never gets removed/released (even if no object is in front of the sensor). These pointers will keep in press state and significantly decrease the fps. 

Has anyone experience this issure before? Is there any workaround to release/cancel all pointers manually? Thx

![6](https://user-images.githubusercontent.com/42322486/97114428-6cb89200-172b-11eb-95d6-9e454155181e.png)

**Comment by mchangxe** [comment]
experiencing the same problem, did you manage to get it fixed somehow?

**Comment by raymond9991** [comment]
Unfortunately no. I've found that turning the TouchManager gameobject on/off will release some of the pointers but not all, eventually the pointers will still overflow. Still can't manage to find out the reason behind, but the error message "ArgumentException: An item with the same key has already been added." appears a lot before a fps dip. I'm guessing some duplicate pointers with same id is added somehow, don't know why, and can't find a workaround...

**Comment by dhr0118** [comment]
> Unfortunately no. I've found that turning the TouchManager gameobject on/off will release some of the pointers but not all, eventually the pointers will still overflow. Still can't manage to find out the reason behind, but the error message "ArgumentException: An item with the same key has already been added." appears a lot before a fps dip. I'm guessing some duplicate pointers with same id is added somehow, don't know why, and can't find a workaround...

@raymond9991  hi, did you solve the problem later?

**Comment by ouroeight** [comment]
> 
> 
> > Unfortunately no. I've found that turning the TouchManager gameobject on/off will release some of the pointers but not all, eventually the pointers will still overflow. Still can't manage to find out the reason behind, but the error message "ArgumentException: An item with the same key has already been added." appears a lot before a fps dip. I'm guessing some duplicate pointers with same id is added somehow, don't know why, and can't find a workaround...
> 
> @raymond9991 hi, did you solve the problem later?

No, no solution. My workaround is simply turn on/off the application when it becomes very laggy...

**Comment by valcion1993** [comment]
I had a similar problem, and in my case it was that the touch manager was not configured correctly.

Go to an example scene and paste the touch manager again
